plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/file/FileHiveMetastore.java

```diff
@@ -560,7 +560,7 @@ public synchronized void renameTable(HiveIdentity identity, String databaseName,
         getRequiredDatabase(newDatabaseName);
 
         if (isIcebergTable(table.getParameters())) {
-            throw new TrinoException(NOT_SUPPORTED, "Rename not supported for Iceberg tables");
+            throw new TrinoException(NOT_SUPPORTED, "This connector does not support renaming tables");
```

Comment:
the previous wording was more correct than the current one.  If Iceberg doesn't support rename, it should be handled in IcebergMetadata.
If Iceberg does support rename (delegating to the connector), we shouldn't "lie" it does not.

Reply:
I'll revert